### PR TITLE
[PLATFORM-1403] Updated IMS authorize and token endpoints

### PR DESCRIPTION
**Pull request by jwixn**
## Description

Updated IMS authorize and token endpoints in the sample Postman collection  to use latest version of each

## Motivation and Context

Our samples should demonstrate calling the latest version of all of our endpoints.

## How Has This Been Tested?
Tested the updated Postman collection locally and found no regressions

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by jwixn** [comment]
It's a trivial change, but we can wait until after MAX to merge, so there is no rush.

**Review by tmartin8080 — APPROVED** [review]

**Review by reagrawa — COMMENTED** [review]
LGTM

**Pull request merged**
